hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -192,30 +192,28 @@ The [HDMI boot diagnostics](./boot_diagnostics.md) display is disabled if DISABL
 Default: 0   
 Version: pieeprom-2020-04-16.bin  
 
-`Version: pieeprom-2020-06-15.bin - STABLE`
+`Version: pieeprom-2020-09-03`
 The `disable_splash` property is no longer checked because the HDMI diagnostics screen is started before config.txt is read.
 
 ### ENABLE_SELF_UPDATE
-Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set `ENABLE_SELF_UPDATE=1` and add `bootloader_update=1` in config.txt.
-N.B. There is no automatic rollback in the event of a power failure during the firmware update. In the unlikely event of this happening you will have to use Pi Imager to apply the rescue image.
+Enables the bootloader to update itself from a network or USB-MSD boot.
 
-If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
+If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the USB MSD boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
```

Comment:
Does the configuration within the EEPROM image (i.e. bootconf.txt) also count towards the "update files being different"?

Reply:
Updated, the Raspberry Pi Imager does warn about the reformat, but ...

Reply:
Yes, it's a binary diff of the entire EEPROM image